src/main/java/seedu/address/storage/XmlAdaptedRoom.java

```diff
@@ -0,0 +1,148 @@
+package seedu.address.storage;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.xml.bind.annotation.XmlElement;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.person.Guest;
+import seedu.address.model.room.Capacity;
+import seedu.address.model.room.DoubleRoom;
+import seedu.address.model.room.Expenses;
+import seedu.address.model.room.Reservations;
+import seedu.address.model.room.Room;
+import seedu.address.model.room.RoomNumber;
+import seedu.address.model.room.SingleRoom;
+import seedu.address.model.room.SuiteRoom;
+import seedu.address.model.tag.Tag;
+
+/**
+ * JAXB-friendly version of the Room.
+ */
+public class XmlAdaptedRoom {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Room's %s field is missing!";
+
+    @XmlElement(required = true)
+    private String roomNumber;
+    @XmlElement(required = true)
+    private String capacity;
+
+    @XmlElement
+    private String expenses;
+    @XmlElement
+    private String reservations;
```

Comment:
expenses should be a List<Expense>. I think reservations should be a List also, check with whoever doing XML adaptation for reservations. But i think can merge this first, i'm going to edit this in my PR.

Reply:
Okay thanks thanks! Strings were just for placeholder so I can convert them to something to display on XML so I can test out listing. Yes can make changes next time.